mails/themes/classic/core/backoffice_order.html.twig

```diff
@@ -0,0 +1,34 @@
+{% extends '@MailThemes/classic/components/layout.html.twig' %}
+
+{% block content %}
+<tr>
+  <td align="center" class="titleblock">
+    <font size="2" face="{{ languageDefaultFont }}Open-sans, sans-serif" color="#555454">
+      <span class="title">{{ 'Hi {firstname} {lastname},'|trans({}, 'EmailsBody', locale)|raw }}</span>
+    </font>
+  </td>
+</tr>
+<tr>
+  <td class="space_footer">&nbsp;</td>
+</tr>
+<tr>
+  <td class="box" style="border:1px solid #D6D4D4;">
+    <table class="table">
+      <tr>
+        <td width="10">&nbsp;</td>
+        <td>
+          <font size="2" face="{{ languageDefaultFont }}Open-sans, sans-serif" color="#555454">
+            <p style="border-bottom:1px solid #D6D4D4;">
```

Comment:
we have `languageDefaultFont`, can we have `defautBorderColor` because #D6D4D4 is used many times in many templates.

Reply:
In this version we don't worry too much about template customization, this will come in later issues (probably not before 1.7.7)
But `languageDefaultFont` is a special case as it is used to include the `Tahoma` font for arabic languages for example.